src/components/Tipseen/Tipseen.tsx

```diff
@@ -55,7 +55,7 @@ interface TipseenProps extends VibeComponentProps {
    */
   isCloseButtonOnImage?: boolean;
   closeButtonOnImage?: boolean;
-  onClose?: () => void;
+  onClose?: (event: React.MouseEvent) => void;
```

Comment:
@meromco can you make it more specific to `React.MouseEvent<HTMLButtonElement>`?
also, please make the `event` optional, as otherwise it would be a breaking change

```suggestion
  onClose?: (event?: React.MouseEvent<HTMLButtonElement>) => void;
```